### PR TITLE
Fix replying from search results for this and all rooms

### PR DESCRIPTION
**Pull request by t3chguy**
Fixes https://github.com/vector-im/riot-web/issues/10796

Signed-off-by: Michael Telatynski <7t3chguy@gmail.com>

### CODE REVIEW

File: src/components/structures/RoomView.js [hunk 1]
```diff
@@ -586,7 +586,6 @@ module.exports = createReactClass({
                 return ContentMessages.sharedInstance().sendContentListToRoom(
                     [payload.file], this.state.room.roomId, MatrixClientPeg.get(),
                 );
-                break;
```
Comment (turt2live) [inline src/components/structures/RoomView.js]: this seems unrelated?

Comment (t3chguy) [reply src/components/structures/RoomView.js]: It is, but its just a small delint, saw it out of the corner of my eye and couldn't help myself, its a break after a return

Comment (turt2live) [reply src/components/structures/RoomView.js]: oh, I didn't even notice that. It should probably drop the return and break instead, unless the return is doing something.

Comment (t3chguy) [reply src/components/structures/RoomView.js]: oh thats an even better point, onAction is a void method iirc
